game-core/src/main/java/games/strategy/triplea/delegate/battle/steps/BattleStep.java

```diff
@@ -75,6 +75,9 @@
     OFFENSIVE_GENERAL_RETREAT,
     STALEMATE_BATTLE_END_CHECK,
     SUB_DEFENSIVE_RETREAT_AFTER_BATTLE,
+
+    // Fire Round steps
```

Comment:
Is the comment used to group the enums? 
A bit nitpicky, I'd recommend to avoid it. As a casual reader here, it does not make a ton of sense what the comment is delimiting. It's also a bit of an issue for maintenance, does every line break indicate a new group? Are the groups all up to date?

If needing to mark certain enum values, there should be other ways to do it. For example, one can add name patterns, like a prefix, add flags, or use additional enum types, or convert the enums to strings.